sentencepiece/debian/control

```diff
@@ -0,0 +1,35 @@
+Source: sentencepiece
+Priority: optional
+Maintainer: Kentaro Hayashi <hayashi@clear-code.com>
+Build-Depends:
+ debhelper (>= 9),
+ cmake,
+ pkg-config,
+ libgoogle-perftools-dev
+Standards-Version: 4.1.3
+Section: libs
+Homepage: <insert the upstream URL, if relevant>
+#Vcs-Browser: https://salsa.debian.org/debian/sentencepiece
+#Vcs-Git: https://salsa.debian.org/debian/sentencepiece.git
+
+Package: libgoogle-sentencepiece-dev
```

Comment:
Is `google-` prefix needed?

Reply:
It seems that some packages in debian uses `google-` prefix, but it may be no need to use prefix(not so many packages).
I'll remove it.